spec/integration/plos_details_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper.rb'
+
+describe PlosDetails, :vcr do
+  let(:plos_paper) do
+    create(:plos_paper, id: 1, reader: 0, doi: '10.1371/journal.pone.0089204')
+  end
+  let(:job) {PlosDetails.new}
+
+  it 'updates the view count' do
+    before = plos_paper.reader
+
+    VCR.use_cassette 'plos_detail' do
+      job.perform plos_paper.id
+    end
```

Comment:
You don't need the `use_cassette` block, that's what the `:vcr` on top is doing, deriving the name of the VCR file from the test.